docs/en/juicefs_on_windows.md

```diff
@@ -51,7 +53,7 @@ ws.Run "C:\juicefs\juicefs.exe mount redis://192.168.1.8:6379/1 Z:",0
 
 ### 2. Set boot up
 
-Use the keyboard combination `win + R` to call up the running window
+Use the keyboard combination <kbd>Windows + R</kbd> to call up the running window
 
 ![Windows run](../images/windows-run-startup.png)
 
```

Comment:
The link in line 66 is broken (GitHub doesn't allow comment on that line, sorry)